src/main/java/com/exasol/adapter/dialects/JdbcAdapterProperties.java

```diff
@@ -0,0 +1,62 @@
+package com.exasol.adapter.dialects;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import com.exasol.adapter.AdapterProperties;
+import com.exasol.errorreporting.ExaError;
+
+/**
+ * Special Adapter Properties for JDBC-based Virtual Schemas.
+ */
+public class JdbcAdapterProperties {
```

Comment:
Having this as a static class absolutely breaks the design. Please either embed this into the using class or make it a regular class. Even if that means we break the interface, that would be a lot cleaner.

To be honest, I'd rather break the interface here. If we are honest, we have to update the dependent VS anyway, so we might as well do it right.